chalicelib/dcc/transformer_mapping.json

```diff
@@ -7,47 +7,28 @@
       },
       {
         "source": "metadata.json",
-        "index_field": "analysis_type",
```

Comment:
Let's keep "analysis_type" because the dashboard service expects it to be there and I think we will likely be able to get values for this at some point in the future. Let's make it an empty string constant for now.